packages/node_modules/@webex/plugin-meetings/src/meeting-info/meeting-info-v2.js

```diff
@@ -77,6 +77,59 @@ export default class MeetingInfoV2 {
     });
   }
 
+  /**
+     * Fetches meeting info from the server
+     * @param {String} conversationUrl one of many different types of destinations to look up info for
+     * @returns {Promise} returns a meeting info object
+     * @public
+     * @memberof MeetingInfo
+     */
+  async createAdocSpaceMeeting(conversationUrl) {
+    const getInvitees = (particpants = []) => {
+      const invitees = [];
+
+      if (particpants) {
+        particpants.forEach((participant) => {
+          invitees.push({
+            email: participant.emailAddress,
+            ciUserUuid: participant.entryUUID
+          });
+        });
+      }
+
+      return invitees;
+    };
+
+    return this.webex.internal.conversation.get(
+      {url: conversationUrl},
+      {includeParticipants: true, disableTransform: true}
+    )
+      .then((conversation) => {
+        const body = {
+          title: conversation.displayName,
```

Comment:
I guess this is correct, but would be good to double check if the thick clients are setting the same title when starting adhoc meetings as this should be consistent across all clients

Reply:
checked with adbullah 
<img width="1244" alt="Screen Shot 2022-04-06 at 1 05 46 PM" src="https://user-images.githubusercontent.com/3895486/162029430-dc932e7a-a36e-4a6c-b9fe-7cc59c0131ab.png">


They do some logic on the thick client for the obtp but they only send the conversation